packages/wonder-blocks-modal/components/modal-launcher-positioner.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+
+type Props = {
+    children: React.Node,
+    closeModal: () => void,
+};
+
+/**
+ * A private component used by ModalLauncher. This is the fixed-position
+ * container element that gets mounted outside the DOM. It overlays the modal
+ * content (provided as `children`) over the content, with a gray backdrop
+ * behind it.
+ */
+export default class ModalLauncherPositioner extends React.Component<Props> {
+    /**
+     * When the user clicks on the gray backdrop area (i.e., the click came
+     * _directly_ from the positioner, not bubbled up from its children), close
+     * the modal.
+     */
+    _handleClick = (e: SyntheticEvent<>) => {
+        // Was the lowest-level click target (`e.target`) the positioner element
+        // (`e.currentTarget`)?
+        if (e.target === e.currentTarget) {
+            this.props.closeModal();
+        }
+    };
+
+    render() {
+        return (
+            // TODO(mdr): Using a div instead of a View for now, because we
+            //     haven't developed our click handling story for View yet.
+            <div
+                className={css(styles.modalPositioner)}
+                onClick={this._handleClick}
+            >
```

Comment:
`<View style={styles.modalPositioner}>`

You'll have to update `View` to pass through the `onClick` handler.  I would just pass through all other props.  Sorry for not catching this in the previous review.

Reply:
Gotcha, this was @jeresig's advice since he wasn't sure what the click story was for `View`s. If `onClick` is the plan though, I'm ready to go ahead with it!